@nav-frontend/react-accordion/src/Accordion.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { forwardRef, useEffect, useRef, useState } from "react";
+import cls from "classnames";
+import {
+  UnmountClosed,
+  Collapse,
+  CollapseProps,
+  CollapseCallbackArgs,
+} from "react-collapse";
+
+import { Expand } from "@nav-frontend/icons";
+import "@nav-frontend/accordion-styles";
+import { guid } from "nav-frontend-js-utils";
+
+interface AccordionProps {
+  children: React.ReactNode;
+  title: React.ReactNode;
+  open?: boolean;
+  onClick?: (event: React.SyntheticEvent<HTMLButtonElement>) => void;
+  className?: string;
+  renderContentWhenClosed?: boolean;
+  collapseProps?: Partial<CollapseProps>;
```

Comment:
Ved å eksponere `collapseProps` låser vi oss til react-collapse. Jeg gjorde et raskt søk, og det ser ut som den eneste bruken i navikt av `collapseProps` er for å sette `hasNestedCollapse`, og den er deprecated (https://github.com/nkbt/react-collapse#3-deprecated-props-not-available-in-v5). Hva tenker du om å fjerne denne fra AccordionProps, og heller legge den på igjen om behoved dukker opp?

Reply:
Enig, fjernet collapseProps. La til onRest direkte da dette er den har mest usecases, men trenger vi denne i det hele tatt?

Reply:
Næh, la oss droppe den